### PR TITLE
Rephrase "import dump"

### PR DESCRIPTION
**Pull request by amire80**
"ImportDump" is OK as an extension name,
but in messages, "import dump" is confusing.
Here, the word "dump" is removed from most
messages, and "import dump request" is changed
to simply "import request".

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -4,18 +4,18 @@
 			"Universal Omega"
 		]
 	},
-	"action-handle-import-dump-interwiki": "handle import dump interwiki",
+	"action-handle-import-dump-interwiki": "handle interwiki dump import",
```
Comment (Universal-Omega) [inline i18n/en.json:7]: This doesn't make sense to me. This isn't to handle an "interwiki dump" but rather to add interwiki prefixes to the remote wiki for --username-prefix,  but to me this makes it seem like we are importing an interwiki dump.

File: i18n/en.json [hunk 2]
```diff
@@ -122,20 +122,20 @@
...
+	"logentry-importdumpprivate-request": "$1 {{GENDER:$2|requested}} an import for private wiki $4 in $5.",
+	"logentry-importdumpprivate-started": "$1 {{GENDER:$2|started}} an import for private wiki $4 in $5.",
+	"logentry-importdumpprivate-statusupdate": "$1 {{GENDER:$2|updated}} status of private import request $4 to '$5'.",
 	"requestimportdump": "Request import",
 	"requestimportdumpqueue": "Import requests queue",
 	"requestimportdumpqueue-header": "Search import requests",
 	"requestimportdumpqueue-header-info": "You can use this form to search through import requests. Use the parameters below to refine your search.",
-	"right-handle-import-dump-interwiki": "Handle import dump interwiki",
-	"right-handle-import-dump-requests": "Handle user import dump requests",
+	"right-handle-import-dump-interwiki": "Handle interwiki dump import",
```
Comment (Universal-Omega) [inline i18n/en.json:137]: Same thing here

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: i18n/en.json (hunk 1 above)
Comment (amire80) [reply i18n/en.json:7]: Oh, got it. How about "handle interwiki prefixes in dump imports"?

Comment (Universal-Omega) [reply i18n/en.json:7]: That's better, still a bit confusing, I was thinking maybe like "handle remote interwiki prefixes for import requests" or something (not exact and that may be to long also, overall I do agree with your wording changes here though.

**Review by Universal-Omega — APPROVED** [review]

**Pull request merged**